core/src/main/java/brooklyn/enricher/basic/AbstractAggregator.java

```diff
@@ -187,6 +187,24 @@ protected void setEntityAfterSubscribingProducerChildrenEvents() {
         }
     }
 
+    /** true if this should aggregate members */
+    protected boolean isAggregatingMembers() {
+        if (Boolean.TRUE.equals(fromMembers)) return true;
```

Comment:
remind me: is setEntity(EntityLocal) called on rebind? OR could `fromMembers` potentially be null after rebeind?